tests/test_agent.py

```diff
@@ -0,0 +1,71 @@
+import gym
+
+# TEMPORARY, ONLY USED TO MAKE THIS TEST WORK FOR NOW
+import sys
+
+sys.path.append("../dumbbells")
+from dumbbells.Agent import Agent
+
+
+def testQAgent():
+    print("\nStarting testQAgent")
```

Comment:
Since we're using `pytest` there's no need to log tests individually, they'll be logged by `pytest` itself